{"name": "Проверка создания пользователя по username, длина которого меньше допустимого (< 6 символов)", "status": "failed", "statusDetails": {"message": "AssertionError: Код ответа приложения должен быть 400\nassert 210 == 400\n  +210\n  -400", "trace": "self = <test_api.TestApiAuthorizedUser object at 0x7f9070dcd280>\ncreate_user_data_with_middle_name = {'email': 'n4QASX8@ya.ru', 'middle_name': 'JennaRichardson', 'name': 'Jenna', 'password': 'VOO8SfyscLCB', ...}\n\n    @allure.title(\"Проверка создания пользователя по username, длина которого меньше допустимого (< 6 символов)\")\n    @pytest.mark.API\n    def test_post_create_user_invalid_data_username_length_less_than_acceptable_bug(self,\n                                                                                    create_user_data_with_middle_name):\n        \"\"\"\n         Тестирование: Добавление пользователя с неккоректными данными\n         Эндпоинт: POST http://127.0.0.1:8086/api/user\n         Предусловия: Авторизация, создание пользовательских данных, заменить username на слово, длиной меньше\n         допустимого (меньше чем 6 символов)\n         Шаги:\n         1. Отправка запроса по API\n         Ожидаемый результат:\n         Код ответа - 400 (\"BAD REQUEST\") (-)\n         Пользователь НЕ добавляется в БД (-)\n         Фактический результат:\n         Код ответа - 210 (\"UNKNOWN\")\n         Пользователь добавляется в БД\n         \"\"\"\n        complex_user_data = create_user_data_with_middle_name\n        complex_user_data[\"username\"] = RandomGenerate.generate_random_user_name(1, 5)\n    \n        response = self.post_create_user(complex_user_data)\n        self.mysql.session.commit()\n        user_db = self.mysql.session.query(Users).filter_by(email=complex_user_data['email']).first()\n    \n>       assert response.status_code == 400, \"Код ответа приложения должен быть 400\"\nE       AssertionError: Код ответа приложения должен быть 400\nE       assert 210 == 400\nE         +210\nE         -400\n\ncomplex_user_data = {'email': 'n4QASX8@ya.ru', 'middle_name': 'JennaRichardson', 'name': 'Jenna', 'password': 'VOO8SfyscLCB', ...}\ncreate_user_data_with_middle_name = {'email': 'n4QASX8@ya.ru', 'middle_name': 'JennaRichardson', 'name': 'Jenna', 'password': 'VOO8SfyscLCB', ...}\nresponse   = <Response [210]>\nself       = <test_api.TestApiAuthorizedUser object at 0x7f9070dcd280>\nuser_db    = <test_users(id='39',username='vXu',name='Jenna',surname='Richardson',surname='JennaRichardson',password='VOO8SfyscLCB',email='n4QASX8@ya.ru',access='1',active='0',start_active_time='None')>\n\ncode/api_myapp/test/test_api.py:372: AssertionError"}, "description": "\n         Тестирование: Добавление пользователя с неккоректными данными\n         Эндпоинт: POST http://127.0.0.1:8086/api/user\n         Предусловия: Авторизация, создание пользовательских данных, заменить username на слово, длиной меньше\n         допустимого (меньше чем 6 символов)\n         Шаги:\n         1. Отправка запроса по API\n         Ожидаемый результат:\n         Код ответа - 400 (\"BAD REQUEST\") (-)\n         Пользователь НЕ добавляется в БД (-)\n         Фактический результат:\n         Код ответа - 210 (\"UNKNOWN\")\n         Пользователь добавляется в БД\n         ", "steps": [{"name": "Выполняем POST запрос '/api/user'", "status": "passed", "steps": [{"name": "Выполняем запрос", "status": "passed", "parameters": [{"name": "method", "value": "'POST'"}, {"name": "location", "value": "'/api/user'"}, {"name": "headers", "value": "{'Content-Type': 'application/json'}"}, {"name": "data", "value": "None"}, {"name": "params", "value": "None"}, {"name": "json", "value": "{'username': 'vXu', 'password': 'VOO8SfyscLCB', 'email': 'n4QASX8@ya.ru', 'name': 'Jenna', 'surname': 'Richardson', 'middle_name': 'JennaRichardson'}"}], "start": 1653172682366, "stop": 1653172682390}], "parameters": [{"name": "complex_data", "value": "{'username': 'vXu', 'password': 'VOO8SfyscLCB', 'email': 'n4QASX8@ya.ru', 'name': 'Jenna', 'surname': 'Richardson', 'middle_name': 'JennaRichardson'}"}], "start": 1653172682365, "stop": 1653172682390}], "start": 1653172682365, "stop": 1653172682402, "uuid": "c6372742-b6ba-42a7-8012-5e06aaf3258e", "historyId": "af87bbdb012ad17b97db87632dde3b84", "testCaseId": "cfd745f69f9bbb51d71d3601cfff5e34", "fullName": "code.api_myapp.test.test_api.TestApiAuthorizedUser#test_post_create_user_invalid_data_username_length_less_than_acceptable_bug", "labels": [{"name": "tag", "value": "API"}, {"name": "parentSuite", "value": "code.api_myapp.test"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApiAuthorizedUser"}, {"name": "host", "value": "65c64318f096"}, {"name": "thread", "value": "1-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "code.api_myapp.test.test_api"}]}